phoenix-pherf/README.md

```diff
@@ -0,0 +1,105 @@
+Pherf is a performance test framework that exercises HBase through Apache Phoenix, a SQL layer interface.
```

Comment:
This readme is nice. Would you mind creating a markdown version of it for our site and adding a menu item under Using (in site.xml) that points to it? See the About menu for a link for how to update the website.